app/models/concerns/host_common.rb

```diff
@@ -32,7 +32,6 @@ module HostCommon
     has_many :lookup_values, :primary_key => :lookup_value_matcher, :foreign_key => :match, :dependent => :destroy
     # See "def lookup_values_attributes=" under, for the implementation of accepts_nested_attributes_for :lookup_values
     accepts_nested_attributes_for :lookup_values
-    has_many :lookup_keys, :through => :lookup_values
```

Comment:
I should say that I looked at this when I was merging #6007 and thought it was a harmless, but potentially useful association.  I guess it's unused right now?